src/amber/cli/templates/app/config/webpack/common.js

```diff
@@ -20,20 +19,13 @@ let config = {
   module: {
     rules: [
       {
-        test: /\.css$/,
+        test: /\.(sa|sc|c)ss$/,
```

Comment:
`/\.(css|sass|scss)$/` plz

Reply:
Agreed! I'd lifted this from the `mini-css-extract-plugin` readme but I prefer yours!